specs/Storage.Pickers/FileSavePicker.md

```diff
@@ -0,0 +1,195 @@
+FileSavePicker Class
+===
+
+# Background
+
+Namespace: [Microsoft.Windows.Storage.Pickers](./Microsoft.Windows.Storage.Pickers.md)
+
+Represents a UI element that lets the user choose a file to save.
+
+# API Pages
+
+## Definition
+
+```C#
+runtimeclass FileSavePicker
+{
+    FileSavePicker(Microsoft.UI.WindowId windowId);
+
+    string CommitButtonText;
+    string DefaultFileExtension;
+    string SuggestedFileName;
+    string SuggestedSaveFilePath;
```

Comment:
We had discussed TrySuggestedSaveFilePath overriding SuggestedFileName.  But seeing these two properties now side by side, the redundancy is jarring.  I propose that we remove SuggestedFileName and just keep the existing parsing behavior that we had for TrySuggestedSaveFilePath, applied to SuggestedSaveFilePath.  I.e., you can provide a folder and optionally a file name to initialize the dialog with.  We could of course deterministically define the behavior of these two properties, such as "last one wins" - but it just seems like a confusing design to have both.